{
  "schema_version": "1.2.0",
  "id": "GHSA-m269-wj6g-c459",
  "modified": "2022-06-11T03:26:10Z",
  "published": "2022-05-17T02:56:23Z",
  "aliases": [
    "CVE-2016-10127"
  ],
  "summary": "PySAML2 XML external entity attack",
  "details": "PySAML2 allows remote attackers to conduct XML external entity (XXE) attacks via a crafted SAML XML request or response.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.0/AV:N/AC:L/PR:L/UI:R/S:C/C:H/I:H/A:H"
    }
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pysaml2"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "4.5.0"
            }
          ]
        }
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2016-10127"
    },
    {
      "type": "WEB",
      "url": "https://github.com/rohe/pysaml2/issues/366"
    },
    {
      "type": "WEB",
      "url": "https://github.com/rohe/pysaml2/pull/379"
    },
    {
      "type": "WEB",
      "url": "https://github.com/rohe/pysaml2/commit/6e09a25d9b4b7aa7a506853210a9a14100b8bc9b"
    },
    {
      "type": "WEB",
      "url": "https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=850716"
    },
    {
      "type": "WEB",
      "url": "http://www.openwall.com/lists/oss-security/2017/01/19/5"
    },
    {
      "type": "WEB",
      "url": "http://www.securityfocus.com/bid/95376"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-611"
    ],
    "severity": "CRITICAL",
    "github_reviewed": false
  }
}